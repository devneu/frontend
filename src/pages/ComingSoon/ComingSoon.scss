body{

  &:before {
    content: "";
    position: fixed;
    top: 0;
    left: 0;
    height: 100%;
    width: 100%;
    background-color: #2a5555;
    opacity: 0.7;
    z-index: 1;
  }
}

.bg-video {
  position: fixed;
  right: 0;
  bottom: 0;
  min-width: 1920px;
  min-height: 1080px;
  z-index: 0;
}

.coming-soon-header {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
}

.form-block-coming-soon {
  height: 100vh;
  width: 70%;
  padding: 50px 40px;
  transform: skewX(-9deg);
  transform-origin: top right;
  z-index: 1;
  background-color: darkblue;
  opacity: 0.9;
  h1 {
    font-size: 3.5rem;
    margin: 50px 40px;
    color: #fff;
  }

  p {
    margin: 0 40px 0 40px;
    font-size: 1.2rem;
    color: #fff;
  }
}

.form {
  box-sizing: border-box;

  padding-left: 150px;
  padding-top: 150px;

  input {
    padding: 15px;
    width: 300px;
  }

  button {
    height: 54px;
    margin-left: 20px;
    padding: 1rem;
  }
}

.social-links {
  display: flex;
  flex-direction: column;
  align-items: flex-end;
  margin-right: 20px;
  height: 100vh;
  width: 28%;
  justify-content: center;
  z-index: 2;
}

.social {
  height: 50px;
  width: 50px;
  border-radius: 100%;
  margin: 20px;
  cursor: pointer;

  img {
    height: 50px;
    width: 50px;
    z-index: 5;

  }
}
