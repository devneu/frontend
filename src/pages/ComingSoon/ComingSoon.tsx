import React from 'react';
import './ComingSoon.scss';
import { Input, Button, Typography } from 'antd';
import ReactPlayer from 'react-player';

const { Title, Paragraph } = Typography;

const ComingSoon = () => (
  <div className="coming-soon">
    <ReactPlayer className="bg-video" url="https://youtu.be/XGw3OgCBo_U" loop muted playing />
    <div className="content">
      <Title className="title">Our Website is Coming Soon</Title>
      <Paragraph className="text">
        We&apos;re working hard to finish the development of the site. Sign up below to receive
        updates and to be noticed when we launch!
      </Paragraph>
      <form className="subscribe-form">
        <Input type="large" placeholder="Enter Email address..." />
        <Button type="primary">NOTIFY ME!</Button>
      </form>
    </div>
    <nav className="social-links">
      <a className="social-link" href="#twitter">
        <img
          className="social-icon"
          src={`${process.env.PUBLIC_URL}/images/twitter.png`}
          alt="twitter"
        />
      </a>
      <a className="social-link" href="#telegram">
        <img
          className="social-icon"
          src={`${process.env.PUBLIC_URL}/images/telegram.png`}
          alt="telegram"
        />
      </a>
    </nav>
  </div>
);

export default ComingSoon;
