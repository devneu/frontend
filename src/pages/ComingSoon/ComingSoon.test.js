import React from 'react';
import renderer from 'react-test-renderer';
import ComingSoon from './ComingSoon';

describe('ComingSoon', () => {
    it('Should successfully render', () => {
        const wrapper = renderer
            .create(
                <ComingSoon />
            )
            .toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});