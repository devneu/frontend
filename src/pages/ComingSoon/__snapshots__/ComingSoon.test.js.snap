// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComingSoon Should successfully render 1`] = `
<div
  className="coming-soon-page"
>
  <div
    className="bg-video"
    style={
      Object {
        "height": "360px",
        "width": "640px",
      }
    }
  />
  <header
    className="coming-soon-header"
  >
    <div
      className="subscribe-form"
    >
      <h1
        className="ant-typography"
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Our Website is Coming Soon
      </h1>
      <p>
        We apos; re working hard to finish the development of the site. Sign up below to receive updates and to be noticed when we launch!
      </p>
      <div
        className="row-input-mail"
      >
        <input
          className="ant-input"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Enter Email address..."
          type="large"
          value=""
        />
        <button
          className="ant-btn ant-btn-primary"
          onClick={[Function]}
          type="button"
        >
          <span>
            NOTIFY ME!
          </span>
        </button>
      </div>
    </div>
    <div
      className="social-links"
    >
      <div
        className="social-link"
      >
        <a
          href="#twitter"
        >
          <img
            alt="twitter"
            src="twitter.png"
          />
        </a>
      </div>
      <div
        className="social-link"
      >
        <a
          href="#telegram"
        >
          <img
            alt="telegram"
            src="telegram.png"
          />
        </a>
      </div>
    </div>
  </header>
</div>
`;
